.travis.yml

```diff
@@ -0,0 +1,30 @@
+language: cpp
+compiler: gcc
+sudo: require
+dist: trusty
+
+script:
+  - ./autogen.sh # Undocumented
+  - ./configure --prefix=/usr
+  - make -j$(nproc)
+  - make install DESTDIR=$(readlink -f appdir) ; find appdir/
+  - rm -rf appdir/usr/include/ appdir/usr/lib/pkgconfig/ appdir/usr/lib/*.{a,la} # Don't bundle developer stuff
+  - sudo mv appdir/usr/lib/* /usr/local/lib/ # Workaround for https://github.com/probonopd/linuxdeployqt/issues/160
+  - mkdir -p appdir/usr/share/icons/hicolor/128x128/apps ; cp dist/qpdf.png appdir/usr/share/icons/hicolor/128x128/apps/
```

Comment:
I've been thinking for a while that qpdf needs a logo or an icon or something. I'll put some thought into it. I guess since the icon is actually here in the qpdf repository, I can change it if/when I come up with something. If I were to design my own logo, I would probably create an SVG file for it. Can that be used, or would I need to generate the right resolution png file (which I can definitely do)?